src/leveled_sst.erl

```diff
@@ -2664,8 +2657,8 @@ fetch_value([Pos|Rest], BlockLengths, Blocks, Key, PressMethod) ->
     {BlockNumber, BlockPos} = revert_position(Pos),
     {Offset, Length} = block_offsetandlength(BlockLengths, BlockNumber),
     <<_Pre:Offset/binary, Block:Length/binary, _Rest/binary>> = Blocks,
-    RawBlock = deserialise_block(Block, PressMethod),
-    case fetchfrom_rawblock(BlockPos, RawBlock) of 
+    R = fetchfrom_rawblock(BlockPos, deserialise_block(Block, PressMethod)),
```

Comment:
this could be a spawn_check_block as well.... but chosen not to?

Reply:
Thee was a volume test comparing the options of spawning in both cases or just the other case - and there was no benefit from spawning here.  In this case the Block Index Cache is not yet built, and once that cache is built there will be a hibernate - so we wait for the hibernate instead.